src/ci/docker-compose/cromwell-test/Dockerfile

```diff
@@ -3,5 +3,3 @@ FROM ubuntu:focal
 WORKDIR /cromwell-test/
 COPY docker-setup.sh ./
 RUN ./docker-setup.sh
-USER hoggett
-WORKDIR /home/hoggett
```

Comment:
If we're not going to fight with using this non-root user, let's also remove the sibling [creation of the user](https://github.com/broadinstitute/cromwell/blob/61/src/ci/docker-compose/cromwell-test/docker-setup.sh#L90-L98) as well?